### PR TITLE
fix: adjust autosave interval

### PR DESCRIPTION
**Pull request by cwillisf**
### Resolves

Resolves ENG-74

### Proposed Changes

Adjust autosave interval from 120 to 600 seconds.

### Reason for Changes

This should still support our users well while reducing server load.

### Test Coverage

Tested locally, but testing in staging will be the "real" test.

### CODE REVIEW

File: src/lib/project-saver-hoc.jsx [hunk 1]
```diff
@@ -398,7 +398,7 @@ const ProjectSaverHOC = function (WrappedComponent) {
         vm: PropTypes.instanceOf(VM).isRequired
     };
     ProjectSaverComponent.defaultProps = {
-        autoSaveIntervalSecs: 120,
+        autoSaveIntervalSecs: 600, // 10 minutes = 600 seconds
```
Comment (cwillisf) [inline src/lib/project-saver-hoc.jsx:401]: The comment is mainly here to make it easier to find this line by text search

Comment (AlexisGoodfellow) [reply src/lib/project-saver-hoc.jsx:401]: How dare you make things easy to find? 🤣

**Review by AlexisGoodfellow — APPROVED** [review]
MAKE IT SO!

**Comment by Secret-chest** [comment]
Any thoughts on using `localStorage` for backups and uploading once 15 minutes or so?

**Comment by cwillisf** [comment]
@Secret-chest yes! I can't promise a timeline, of course, but we're certainly thinking about using local storage to support working offline, recovering from browser crashes, and other scenarios like that.

**Pull request merged**